examples/jenkins/pipeline/README.md

```diff
@@ -157,3 +157,29 @@ To run this example:
 14. Start the pipeline again. Go back to step 11 and repeat.
 
 On the first pipeline run, there will be a delay as Jenkins is instantiated for the project. 
+
+## OpenShift Client Plugin Example
+
+The `openshift-client-plugin-pipeline.yaml` build config references a pipeline that showcases the fluent Jenkins pipeline syntax provided by the
```

Comment:
you might point people to the pipeline this is actually executing, in case people can't figure it out by reading the buildconfig (https://github.com/openshift/jenkins-client-plugin/blob/master/examples/jenkins-image-sample.groovy)